unsubscribe.html

```diff
@@ -488,9 +488,21 @@ <h5 id="unsubscribe-intro">{% filter ak_text:"unsubscribe_intro" %}{% endfilter
 			$('input[value="Unsubscribe"]').click();
 		});
 
-
-
-
+		if (actionkit && actionkit.args) {
+			// Preselect a mailing list if query parameter exists
+			if (actionkit.args.mailing_list_id) {
+				var checkboxToSelect = $("input[name=unsub_lists][value='" + actionkit.args.mailing_list_id + "']");
+				if (checkboxToSelect.length > 0) {
+					$(checkboxToSelect[0]).prop('checked', 'true');
+				} else {
+					// If no checkbox with mailing list ID passed through in query parameter, pre-select all mailing lists
+					$("input[name=unsub_lists]").prop('checked', 'true')
+				}
+			} else {
+				// If no mailing list ID passed through in query parameter, pre-select all mailing lists
+				$("input[name=unsub_lists]").prop('checked', 'true')

```

Comment:
i think the "unsub_lists" strings should be surrounded by single quotes? Also, super minor but a couple of lines should have terminating semicolons